packages/utils/README.md

```diff
@@ -50,4 +50,100 @@ export async function getStaticProps(context) {
 
   return { props: { content } }
 }
+```
+
+## extendAsyncProps
+
+`extendAsyncProps` is similar to `getAsyncProps` except it look up the inputs by name across components and map it's props to the result of the asynchronus mapper function passed.
```

Comment:
```suggestion
`extendAsyncProps` is similar to `getAsyncProps` except it looks up the inputs by name across components and map its props to the result of the asynchronus mapper function passed.
```
Also, maybe reword this in simpler terms?

> map its props to the result of the asynchronus mapper function passed.

It might takes a few re-reads to understand properly...